src/app/app.page/app.page.component.html

```diff
@@ -329,16 +329,6 @@ <h4 class="card-title text-left" i18n="@@newLiveStreamCardTitle">
                                                 </ng-container>
                                             </div>
 
-                                                <div class="form-group text-left"  *ngIf="isClusterMode">
```

Comment:
If we cannot show cluster IP addresses list, how can we send it to a specific cluster IP address? Moreover, I cannot see any IP address on the REST request.

Reply:
This is the first phase actually. The second phase is to make the default value of settings.streamFetcherAutoStart false. For the third phase, we will develop another solution to create stream source for specific cluster IP address.